gui/wxpython/iscatt/frame.py

```diff
@@ -249,7 +249,7 @@ def CursorPlotMove(self, x, y, scatt_id):
             x = int(round(x))
             y = int(round(y))
             coords = True
-        except:
+        except (ValueError, TypeError):
```

Comment:
I don't see how valueError could be raised?

Reply:
In NaN of 'inf' values which can be rounded but not converted to int

Reply:
Good point, but I would keep TypeError only, Nan and inf are very unlikely here. Looking at the code, the coordinates can be None, that still results in TypeError.